prespective_rectification/pre_process.py

```diff
@@ -0,0 +1,69 @@
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+from skimage.measure import label
+
+
+def get_largest_cc(segmentation_area):
```

Comment:
What is cc?

Reply:
connected contours, i will clarify the name of the method